roles/kubernetes/node/meta/main.yml

```diff
@@ -15,18 +15,6 @@ dependencies:
     file: "{{ downloads.testbox }}"
     tags: download
   - role: download
-    file: "{{ downloads.netcheck_server }}"
-    when: deploy_netchecker
-    tags: [download, netchecker]
-  - role: download
-    file: "{{ downloads.netcheck_agent }}"
-    when: deploy_netchecker
-    tags: [download, netchecker]
-  - role: download
-    file: "{{ downloads.netcheck_kubectl }}"
-    when: deploy_netchecker
-    tags: [download, netchecker]
-  - role: download
```

Comment:
I'm sorry for confusion, but those must be pre-downloaded for the node role, as the agents daemon sets require them to be present